src/static-data/reporting-config/types.ts

```diff
@@ -114,7 +114,23 @@ export interface NewGitHubIssueLink {
 	labels?: string[]; // "[Pri] High" e.g.
 }
 
-export type TaskLink = SlackLink | P2Link | GeneralLink | NewGitHubIssueLink;
+export interface NewJiraIssueLink {
+	type: 'jira';
+	project: JiraProject;
+	issueType: JiraIssueType;
+}
+
+interface JiraIssueType {
+	name: string;
+	id: number;
+}
+
+interface JiraProject {
+	name: string;
+	id: number;
+}
```

Comment:
**Suggestion: (non-blocking)** What do you think of simplifying this data structure down to just use the ids?

So it would look like:
``` typescript
export interface NewJiraIssueLink {
	type: 'jira';
	projectId: number;
	issueTypeId: number;
}
```

Definitely don't feel strongly here, and mostly want to open up for discussion!

I really like the idea here, but I wonder if it's the right tradeoff for the reporting config. 🤔  I've been trying to keep this data structure as simple as possible to minimize the number of fields people have to fill out to populate the field.

Technically, the only things that really matter are the IDs, as they are used in the URL creation. All the other stuff is just used to add some niceness to the default text.

But... Since that is all manually entered, the name could actually get out of sync with its actual value on the JIRA server. (Like if project with id `123` was called `Foo Project`, but later is updated to `Bar Project`, what we would show in Bugomattic wouldn't match where the URL takes folks).

So it's a small step backwards in terms of user experience, but I think the simplicity might be nice in the long run.

Thoughts?

Reply:
> Technically, the only things that really matter are the IDs, as they are used in the URL creation. All the other stuff is just used to add some niceness to the default text. 

Yes, that's right! I was contemplating between requiring the name for readability vs id only. 

> So it's a small step backwards in terms of user experience, but I think the simplicity might be nice in the long run. Thoughts?

I totally agree with you and I'm okay with simplifying the data structure. It makes sense that it will be an overhead once the project name is updated and we don't want to have that maintenance on the reporting config. 

I appreciate the feedback! I'll get that fixed 👍🏻